pkg/k8s/deployments/translate.go

```diff
@@ -91,6 +91,13 @@ func translate(t *model.Translation) error {
 		devContainer := GetDevContainer(&t.Deployment.Spec.Template.Spec, rule.Container)
 		TranslateDevContainer(devContainer, rule)
 		TranslateOktetoVolumes(&t.Deployment.Spec.Template.Spec, rule)
+		if rule.RunAsUser != nil {
+			t.Deployment.Spec.Template.Spec.SecurityContext = &apiv1.PodSecurityContext{
+				RunAsUser:  rule.RunAsUser,
+				RunAsGroup: rule.RunAsUser,
+				FSGroup:    rule.RunAsUser,
```

Comment:
fsgroup is tricky. I'd not put it if we don't have a specific use case for this. If we do, I'd add it as a separate value as well.